tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -2058,8 +1731,9 @@
     </Shader>
   </ShaderOp>
 
+  <!-- Used by 32-bit atomics and typed 64-bit atomics tests, which require heap descriptor support  -->
   <!-- For explanations of the atomics tests, see comments in and around VerifyAtomicResults in Executiontest.cpp -->
-  <ShaderOp Name="Atomics" PS="PS" VS="VS" CS="CS" AS="AS" MS="MS" TopologyType="TRIANGLE">
+  <ShaderOp Name="AtomicsHeap" PS="PS" VS="VS" CS="CS" AS="AS" MS="MS" TopologyType="TRIANGLE">
```

Comment:
The changes here are mostly removals as well. 64-bit groupshared testing is now in the root variant